pages/repo/edit.js

```diff
@@ -765,6 +760,11 @@ export class EditorPage extends Component {
           >
             {!readOnly && (
               <Sidebar.Tab tabId='edit' label='Editieren'>
+                {!(isNew && !hasUncommittedChanges) && (
+                  <Link route='repo/raw' params={{ repoId, commitId }}>
```

Comment:
The `Uncommitted changes` I need to display who is currently editing the document in the top right corner of the page. We could drop that information completely if you feel like it gets overcomplicated, but I thought we wanted to show that.

Reply:
`!(isNew && !hasUncommittedChanges)` was there to avoid the case of an empty new document but I'll use your trick instead

Reply:
Personally I'd keep the hasUncommitted changes stuff - I believe who is on the document now is interesting context information and it's not thaaat complicated (especially if you compare with edit.js). Plus that way the header remains consistent with the one of the normal editor. Otherwise it feels more like a modal window. But let me know, it's easy to remove.

Reply:
I think loading and showing the `UncommittedChanges` is fine and should be kept. I'm talking about the uncommitted changes mutations—sorry wasn't clear. Let's discuss later.

Reply:
I see. The mutation is useful when a user chooses to edit a doc that didn't have any uncommitted changes. That way, even in this scenario, as soon as the user enters the source code editor, she will be marked as active user. But we could apply the mutation in the md effect and only send it if there was an edit.

Reply:
Ahhh, we'd use `beginChanges` in your solution 💡

Reply:
If think i'll leave this as is... tried with your suggestion but it's creating problem to return to a "not modified" state in the main editor, even if nothing has been changed through the source code editor (since there is something in local storage). Otherwise I need to edit the logic to always compare the local storage content with the last commit content and check if they are identical or not to `beginChanges` or `concludeChanges`